src/components/AwardCard.vue

```diff
@@ -17,7 +17,7 @@
     </div>
     <div
       class="text d-flex flex-column align-center px-10 text-center justify-center"
-      :class="[isCard ? 'white' : 'cream']"
+      :class="[isCard ? 'chocolate background-cream py-5' : 'cream']"
```

Comment:
I think we should be able to use Vuetifys utility classes here `text-surface1`

Reply:
Specifically, Vuetify exposes `bg-{colorName}` and `text-{colorName}` classes out of the box, so we can do this:
```suggestion
      :class="[isCard ? 'bg-surface-2 text-surface-1 py-5' : 'cream']"
```